README.md

```diff
@@ -67,7 +67,22 @@ first time starting the add-on).
 [![Docker Layers][layers-shield]][microbadger]
 [![Docker Pulls][pulls-shield]][dockerhub]
 
-## Configuration
+## Configuration Home assistant
+
+**Note**: _Remember to restart the add-on when the configuration is changed._
+
+**Note2**: _When changing from the official add-on to this one, make sure to change configuration.yaml broker ip. "broker: core-mosquitto" to "broker: localhost"_
```

Comment:
```suggestion
```

Reply:
Didnt remove the entire note, just the last part since that should be obvious.